libs/README.md

```diff
@@ -14,8 +16,40 @@
-| [deer]        | Rust        | [Crates.io](https://crates.io/crates/deer)        | [Docs.rs](https://docs.rs/deer/latest/deer/)               | **Experimental** backend-agnostic deserialization framework, featuring meaningful error messages and context and fail-slow behavior by default |
-| [error-stack] | Rust        | [Crates.io](https://crates.io/crates/error-stack) | [Docs.rs](https://docs.rs/error-stack/latest/error_stack/) | Context-aware error-handling library that supports arbitrary attached user data                                                                |
+| Directory                  | Language(s) | Publication URL                                               | Docs URL                                                   | Description                                                                                                                                          |
+| -------------------------- | ----------- | ------------------------------------------------------------- | ---------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [antsi]                    | Rust        | [Crates.io](https://crates.io/crates/antsi)                   | [Docs.rs](https://docs.rs/antsi/latest/antsi/)             | Supports coloring Select Graphic Rendition (as defined in ISO 6429) with no external dependencies                                                    |
+| [deer]                     | Rust        | [Crates.io](https://crates.io/crates/deer)                    | [Docs.rs](https://docs.rs/deer/latest/deer/)               | **Experimental** backend-agnostic deserialization framework, featuring meaningful error messages and context and fail-slow behavior by default       |
+| [error-stack]              | Rust        | [Crates.io](https://crates.io/crates/error-stack)             | [Docs.rs](https://docs.rs/error-stack/latest/error_stack/) | Context-aware error-handling library that supports arbitrary attached user data                                                                      |
+| [@hashintel/design-system] | TypeScript  | [npm](https://www.npmjs.com/package/@hashintel/design-system) | To be written                                              | Reusable UI primitives                                                                                                                               |
+| [@hashintel/type-editor]   | TypeScript  | [npm](https://www.npmjs.com/package/@hashintel/type-editor)   | To be written                                              | A user interface for editing entity types defined according to the [Block Protocol's Type System](https://blockprotocol.org/docs/working-with-types) |
+
+## Contributing
+
+See [CONTRIBUTING.md](../CONTRIBUTING.md).
+
+## Publishing
+
+### Rust
+
+TODO
```

Comment:
We have a `setup` job, wich checks for publishable crates. If any of those crates has a change in it's version, a job is run in CI. It's a dry-run in PRs and merge queue and a publish job on push to main.

Reply:
How do we mark something as publishable?

Reply:
https://github.com/hashintel/hash/blob/932a23998bbe1b14218d14981aee1ed0b4bbc1a0/.github/scripts/rust/setup.py#L32-L36

Reply:
I'm happy if someone wants to push something to this branch to fill in this section or suggest something here to go in it (or it can be done in follow up if I haven't had anything by the time it's good to be merged – this has been sitting around for a while)

Reply:
```suggestion
Publishable Rust crates are automatically published on merge to `main` if their version has been modified.

A crate can be marked as _publishable_ by adding it to the list of `PUBLISH_PATTERNS` in [.github/scripts/rust/setup.py](./.github/scripts/rust/setup.py)

The publishing process is tested automatically, and therefore can be verified through, a dry-run within CI on pull requests.
```

Reply:
@TimDiekmann any further comments on this suggestion?

Reply:
LGTM 🪨 🚀